.github/pull_request_template.md

```diff
@@ -1,66 +1,83 @@
-### Pull Request Description
+## ☑️ Resolves
+Fixes #<issue-number>
 
+---
 
-### Related Issue
-<!--
-If this PR is related to an issue, put here, if not, remove this block
--->
-Issue Number:
+## 📝 Summary
+Provide a short, clear summary of what this pull request does and why it is needed.
 
-### Pull Request Type
+---
 
-<!--
-Please check the type of change your pull request introduces. Remove all that is unrelated and remove the comment block too, maintaining only the type of your PR:
+<details>
```

Comment:
Would be best to move this block to the end of document.

Your AI prompt removed a lot of code of details too.